src/actor/actor-document.js

```diff
@@ -38,6 +38,11 @@ export class FabulaUltimaActor extends Actor {
   }
 
   modifiedAttribute({baseAttribute, possibleStatusEffects}) {
+    if (!baseAttribute || !this.attributes) {
+      console.error('Invalid baseAttribute or attributes not defined.');
+      return "6"; // Or some default value
+    }
```

Comment:
When do we run into this?

Reply:
Yah this one was odd.  ONLY for the monster sheet, if I don't provide a default value at creation, the newly create monster actor sheet isn't even able to open.  With this error in the console:
```
TypeError: An error occurred while rendering FabulaUltimaMonsterSheet 33. 
Cannot read properties of undefined (reading 'baseDexterity')
    at FabulaUltimaActor.modifiedAttribute (fabula-ultima.js:1:2666)
    at get dexterity [as dexterity] (fabula-ultima.js:1:2116)
    at lookupProperty (handlebars.min.js:27:16153)
    at Object.eval [as main] (eval at createFunctionContext (handlebars.min.js:29:1808), <anonymous>:53:132)
    at c (handlebars.min.js:27:15429)
    at d (handlebars.min.js:27:15744)
    at e (handlebars.min.js:28:22748)
    at Object.h (handlebars.min.js:27:18738)
    at Object.c [as invokePartial] (handlebars.min.js:27:15036)
    at Object.eval [as main] (eval at createFunctionContext (handlebars.min.js:29:1808), <anonymous>:19:28)
```

Happy to make a change elsewhere if there's something I'm missing!